templates/terraform/aws_agent_with_opentelemetry_collector/main.tf

```diff
@@ -2,7 +2,7 @@
 # Copyright 2023 Monte Carlo Data, Inc.
 
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.9.0"
```

Comment:
tested on v1.8.0 to ensure `terraform init` errors
<img width="1129" height="213" alt="image" src="https://github.com/user-attachments/assets/a69b3a51-411e-462e-9789-2721947d13b0" />